### DISCUSSION TITLE
How to refile to heading with agenda template?

### DISCUSSION THREAD

**Discussion by joshpetit** [nvim-orgmode/orgmode#184] [Q&A]
I'm not sure if this is implemented yet, but I'd like to refile a template to a heading. For example having something like this:

```lua
    org_agenda_templates = {
        w = {
            description = 'Work time',
            template = '** Working on  %<%Y-%m-%d>\n%?',
            target = 'refile.org/Work Timesheet'
        },
    },
```

I'm not sure if I just do't have to syntax correct, if it's not implemented yet I can file a feature request.

**Comment by kristijanhusak** [comment]
Does this exist in Emacs orgmode?

**Reply by joshpetit** [reply]
I'm not sure, I've never used emacs 😅

**Reply by rotlaus** [reply]
Emacs orgmode supports cature to a heading. The capture template would look something like this:
```
(setq org-capture-templates
      '(("w" "Work time (file+headline "~/org/refile.org" "Work Timesheet")
         "** Working on %?\n%T")))
```
Also supported is filing to datetrees, the currently clocked item [and more](https://orgmode.org/manual/Template-elements.html).

**Reply by joshpetit** [reply]
Ok so I'm guessing this should be moved to be a feature request. 

Just made it #196
